devel/yamllint/Portfile

```diff
@@ -26,7 +26,11 @@ checksums           rmd160  5b1bec2ac73695302aae0009cdf19429174757ae \
 
 # Keep Python version compatible with ansible-lint
 
-python.default_version 311
+python.default_version 313
+
+variant python${python.default_version} {}
+
+default_variants +python${python.default_version}
```

Comment:
there is no need/point of doing this; please remove these lines.

Reply:
strange, I had to add these to keep it from trying to use python311.  That no longer is occuring in my testing.